src/s1ap/handlers/s1ap_msg_delegator.c

```diff
@@ -646,6 +660,11 @@ int convertInitCtxRspToProtoIe(SuccessfulOutcome_t *msg, struct proto_IE* proto_
 
 					if(s1apErabSetupItem_p->transportLayerAddress.buf != NULL)
 					{
+                        if (s1apErabSetupItem_p->transportLayerAddress.size != sizeof(int)) {
```

Comment:
May need formatting of the code..seems space & tab combination. Could you pls fix it?

Reply:
Fixed :+1:

Reply:
Fixed :+1: